templates/resources.html

```diff
@@ -50,6 +50,7 @@
                     <p class="title is-4">{{res.title}}</p>
                 </div>
                 <span class="tag is-danger">{{res.level}}</span>
+                <span class="tag is-danger">{{res.type}}</span>
```

Comment:
Do you think a different colour could make the standout, the type?
We don't really have a UX plan, but I believe a different colour could help in telling that a "type" is different from "level"

wdyt?

Reply:
Yeah, i also think the same.
I tried some shades of blue that looked good with the reddish color.

1. #0075A2
2. #3273DC
3. #3E8ED0
![Screenshot 2023-02-06 at 2 38 42 PM](https://user-images.githubusercontent.com/34604329/216930819-6063d761-87d9-4309-a65e-30fa08e0cdb4.png)

Reply:
3rd one looks perfect 👌🏽  `#3E8ED0`

Reply:
I have changed the css in the new commit.